userdocs/profiles.dev/docs/author-docs/remote-helm-chart.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+---
+
+# Remote Helm Chart artifacts
+
+In the [Getting started tutorial](/docs/tutorial-basics/create-a-profile),
+we wrote a very simple profile which contained a single artifact.
```

Comment:
-very

I dislike the use of very. :D It's a redundant, overloaded word which conveys no real meaning. Use instead:

> basic profile
> plain profile
> simple profile
> complexn't